Test/nuget.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="corefx" value="https://dotnet.myget.org/F/dotnet-corefxlab/" />
```

Comment:
Consider adding documentation as to why we need this.

Reply:
Done